myuw/test/dao/test_admin.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase
+from myuw.dao import get_netid_of_original_user
+from myuw.dao.admin import is_admin, can_override
+from myuw.test import fdao_gws_override, get_request_with_user
+
+
+@fdao_gws_override
+class TestAdminDao(TestCase):
```

Comment:
Same here as above:

This test doesn't pass on my build, could you override the settings required on the individual test case level?

Reply:
Changed.